packages/app/components/Header/HamburgerButton.tsx

```diff
@@ -0,0 +1,36 @@
+import HamburgerIcon from "../../public/assets/images/hamburger.svg";
+import CrossIcon from "../../public/assets/images/cross.svg";
+import Image from "next/image";
+
+interface HamburgerButtonProps {
+  open: boolean;
+  toggle: () => void;
+}
+
+export default function HamburgerButton({
+  open,
+  toggle,
+}: HamburgerButtonProps) {
+  return (
+    <button
+      className="space-y-1.5 md:invisible px-[10px] py-[10px] rounded-md  bg-[#F3F4F2] active:bg-[#ECEDEB]"
+      onClick={toggle}
+    >
+      {open ? (
+        <Image
+          aria-label="close-menu"
+          alt="close-menu"
+          src={CrossIcon}
+          className="h-[20px] w-[20px] p-[4px]"
```

Comment:
⚠️ avoid custom sizes ⚠️

Also on the image we can use the `height` and `width` property.

Reply:
Sure. :+1: